e2e/tests-dfx/assetscanister.bash

```diff
@@ -131,6 +131,34 @@ check_permission_failure() {
   assert_command curl --fail -vv -H "Accept-Encoding: gzip" http://localhost:"$PORT"/notreally.js?canisterId="$ID"
   assert_not_contains "content-encoding"
   assert_eq "will not compress" "$stdout"
+
+  # show SetAssetProperties works as expected
+  echo '[
+    {
+      "match": "new_file.txt",
```

Comment:
What would you think about setting these properties on one of the files deployed on line 74?  This way we wouldn't need additional deploy/propose/commit steps

Reply:
Ah, right, I haven't registered the first `dfx deploy` (the one that doesn't go through the proposal).

But just to make sure we're on the same page: the properties will have to be attached to `notreally.js` instead of `new_file.txt`, cause `new_file.txt` won't have any `BatchOperationKind::SetAssetProperties` cause its properties will be attached when calling `create_new_asset`. But this should work fine for `notreally.js` cause it's already in the canister

Reply:
That's right, there are a couple assets that will already be in the canister.  But even better, how about adding another asset specifically to be able to set its properties in the deploy?